src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/MacProxy.cs

```diff
@@ -17,11 +17,7 @@ namespace System.Net.Http
 {
     internal sealed class MacProxy : IWebProxy
     {
-        public ICredentials Credentials
-        {
-            get => null;
-            set => throw new NotSupportedException();
-        }
+        public ICredentials Credentials { get; set; }
```

Comment:
fyi: For the Mac, the work left in #26593 would support at least the 'get' operator here once explicit credentials are read from the Mac system settings.

Reply:
Yup, thanks.

I changed this after some tests were failing that were getting/setting WebRequest.DefaultWebProxy's Credentials.  And in looking around at other IWebProxy implementations and the docs, throwing from set_Credentials is not something we do.